src/pages/worklist/WorkListPage.tsx

```diff
@@ -24,54 +32,169 @@ const StyledTabPanel = styled(TabPanel)`
 
 const DOIRequestTab = 'DoiRequestTab';
 const ReportsTab = 'ReportsTab';
+const OwnershipRequestTab = 'OwnershipRequestTab';
 
 const WorkListPage = () => {
   const { t } = useTranslation();
   const user = useSelector((state: RootState) => state.user);
   const [tabValue, setTabValue] = useState(ReportsTab);
+  const [isLoadingDOI, setIsLoadingDOI] = useState(false);
+  const [loadingErrorDOI, setLoadingErrorDOI] = useState<Error>();
+  const [workListDOIs, setWorkListDOIs] = useState<WorklistRequest[]>([]);
+  const [isLoadingReports, setIsLoadingReports] = useState(false);
+  const [loadingErrorReports, setLoadingErrorReports] = useState<Error>();
+  const [workListReports, setWorkListReports] = useState<WorklistRequest[]>([]);
+  const [isLoadingOwnership, setIsLoadingOwnership] = useState(false);
+  const [loadingOwnershipError, setLoadingOwnershipError] = useState<Error | undefined>();
+  const [workListOwnership, setWorkListOwnership] = useState<WorklistRequest[]>([]);
+  const mountedRef = useRef(true);
 
   const handleTabChange = (event: React.ChangeEvent<unknown>, newValue: string) => {
     setTabValue(newValue);
   };
+
+  useEffect(() => {
+    return () => {
+      mountedRef.current = false;
+    };
+  }, []);
+
+  useEffect(() => {
+    const fetchDOis = async () => {
```

Comment:
Som bruker ville jeg heller nesten forventet at listene ble lastet inn ved tab-change. Tror ikke brukerene kommer til å bla alt for mye fram og tilbake slik at det er irriterende. Jeg ser heller på det som en fordel at listen kan reloades ved å hoppe mellom tabs

Reply:
Dette tetter en memory leak, brukeren sitter ikke og venter på at "rapporteringer" har lastet inn før de går på DOI-forespørsler. Det er også slik vi har det på "mine ressurser" visninga.

Reply:
Forøvrig så kjører requestene i parallell så det skal ikke ta ekstra tid for å vente på alle tre tabsene :)

Reply:
Da lar vi det stå og hører hva brukerne synes. Det går på hvordan de oppfatter det.